lib/services/presets.go

```diff
@@ -113,6 +113,13 @@ func NewPresetAccessRole() types.Role {
 				DatabaseLabels:       types.Labels{types.Wildcard: []string{types.Wildcard}},
 				DatabaseNames:        []string{teleport.TraitInternalDBNamesVariable},
 				DatabaseUsers:        []string{teleport.TraitInternalDBUsersVariable},
+				KubernetesResources: []types.KubernetesResource{
```

Comment:
Should we add some way to backfill this allow condition for roles created prior to this commit?

For example, for clusters using v11 or prior versions, when they upgrade to v12 (I'm assuming this will be backported to v12), they won't have this condition.

I was thinking of something like `defaultAllowRules` but for `KubernetesResources`

Reply:
That already happens [here](https://github.com/gravitational/teleport/blob/c616bcb0568ada5896778eed8b0f1459602433af/api/types/role.go#L750)